selfdrive/manager/process_config.py

```diff
@@ -49,7 +49,7 @@ def qcomgps(started, params, CP: car.CarParams) -> bool:
   DaemonProcess("manage_athenad", "selfdrive.athena.manage_athenad", "AthenadPid"),
   NativeProcess("dmonitoringmodeld", "selfdrive/modeld", ["./dmonitoringmodeld"], enabled=(not PC or WEBCAM) and dp_dm, callback=driverview),
   NativeProcess("encoderd", "system/loggerd", ["./encoderd"]),
-  NativeProcess("loggerd", "system/loggerd", ["./loggerd"], onroad=False, callback=logging),
+  #NativeProcess("loggerd", "system/loggerd", ["./loggerd"], onroad=False, callback=logging),
```

Comment:
loggerd is essential for my debugging purposes. I use recorded videos and log files stored on the disk to fine-tune the system.